### ISSUE TITLE
Session storage

### ISSUE THREAD

**Issue by unematiii** [adrianmay/rhaboo#29]
I'm unable to read from perishable after opening a new tab, window.

I have two stores, perishable and persistent on the page. They both work perfectly after I have wiped the cache (including local/sessionStorage). When I try to instantiate them both after the sessionStorage is wiped and something is written into it afterwards, I'm getting exception 

```javascript
Uncaught TypeError: undefined is not a function
```

I narrowed it down to `left_o_pp`, specifically the line

```javascript
 : ( new global[x[0]]( x[0]=='Date'?x[1]:x[1]?Number(x[1]):undefined ) ) 
```
where the `x = ['&800', fill]`. It seems that when you create an instance for perishable, and there is sth in the localStorage, perishable uses `_RHABOO_NEXT_SLOT` value for persistent to create reference to an empty object. However, the pointed slot does not contain `Object` or sth like that, but another reference, e.g.

```javascript
_RHABOO_NEXT_SLOT	803	
_rhaboo_799	&800	
_rhaboo_800	Object;login=801	
_rhaboo_801	$MyUser;token=802	
_rhaboo_802	$993d9fe28c6f2bbb10b7dd57f6b94e0a	
_rhaboo_id.mydomain.com/v0.8/_session	&799	
```

Simply put, after closing a tab (windows etc) and reopening it, I can instantiate perishable, write into it, but after refreshing the page, I cannot instantiate it anymore.
